.github/workflows/cs-lint.yml

```diff
@@ -0,0 +1,70 @@
+name: CS & Lint
```

Comment:
```suggestion
name: PHP CS & Lint
```

Reply:
Alternatively, leave this as "CS & Lint", and rename the lint.yml to "CS & Lint" as well, and then leave the specifics to the job name. See how the actions are reported to see how best to name/group/identify workflows/jobs/steps:

<img width="560" alt="Screenshot 2021-06-23 at 09 48 03" src="https://user-images.githubusercontent.com/88371/123066898-64267180-d408-11eb-974f-e816165291dc.png">